tutorials/preprocessing/plot_80_boxy_processing.py

```diff
@@ -131,6 +136,11 @@
 # #
 # # The raw intensity values are then converted to optical density.
 
+# doesn't work with the new channel types
```

Comment:
another good reason to use the fnirs_raw for the ac data so this still works and we change as little as possible